pkgs/top-level/python-aliases.nix

```diff
@@ -370,7 +370,8 @@ mapAliases ({
   mir_eval = mir-eval; # added 2024-01-07
   mistune_0_8 = throw "mistune_0_8 was removed because it was outdated and insecure"; # added 2022-08-12
   mistune_2_0 = mistune; # added 2022-08-12
-  mitmproxy-wireguard = throw "mitmproxy-wireguard has been removed because it was replaced by upstream with mitmproxy-rs"; # added 2023-11-06
+  mitmproxy = throw "mitmproxy has been removed because it was replaced by mitmproxy-linux and mitmproxy-macos"; # added 2025-01-16
```

Comment:
```suggestion
  mitmproxy-rs = throw "mitmproxy-rs has been removed because it was replaced by mitmproxy-linux and mitmproxy-macos"; # added 2025-01-16
```